dev-libs/libusb/libusb-1.0.21.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A library that provides generic access to USB devices"
+DESCRIPTION="libusb is meant to facilitate the communication with USB hardware.
+
+- portable: Using a single cross-platform API, it provides access to USB \
+devices on Linux, OS X, Windows and OpenBSD.
+- user-mode: No special privilege or elevation is required for the \
+application to communicate with a device.
+- version-agnostic: All versions of the USB protocol, from 1.0 to 3.0 \
+(latest), are supported."
+HOMEPAGE="http://libusb.info/"
+COPYRIGHT="2001 Johannes Erdfelt
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2016 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray
+	2013-2015 Chris Dickens"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/libusb/libusb/archive/v$portVersion.tar.gz"
+SOURCE_FILENAME="libusb-$portVersion.tar.gz"
+CHECKSUM_SHA256="1a5b08c05bc5e38c81c2d59c29954d5916646f4ff46f51381b3f624384e4ac01"
+ARCHITECTURES="x86 x86_64 x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Hmm, we can add x86_gcc2 here ;-)
```
SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

@Vidrep, after you change libusb-1.0.21.recipe, you can:
```
git commit --amend --no-edit libusb-1.0.21.recipe
```
**REMINDER:** You should only use "--amend" if the commit on the top of your git history is the one you are using to update libusb.